### PR TITLE
Extractor Nodes

### PR DESCRIPTION
**Pull request by Jin1L**
### Description

- [x] I've included a concise description of what each node does

### Styleguide

- [x] My node adheres to the [styleguide](https://github.com/flojoy-io/nodes/blob/main/node_styleguide.md) for Flojoy nodes

### Docs

- [x] I've submitted a PR for a documentation page for the new node(s) that contains usage examples (see docs.flojoy.io)

### CODE REVIEW

**Comment by scorinaldi** [comment]
Hey @Jin1L -- I added the [WIP] prefix as it looks like this work is still ongoing. Also, would it be possible to break this into two separate PRs as the change is getting quite large?

**Comment by Jin1L** [comment]
> Hey @Jin1L -- I added the [WIP] prefix as it looks like this work is still ongoing. Also, would it be possible to break this into two separate PRs as the change is getting quite large?

I separated the PR!

File: MANIFEST/open_image.manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+COMMAND:
+  - name: OPEN_IMAGE
+    key: OPEN_IMAGE
+    type: FILE
+    parameters:
+      file_path:
+        type: string
+        default: ""
```
Comment (scorinaldi) [inline MANIFEST/open_image.manifest.yaml:8]: Can you add a newline to the end of these yamls?

**Review by scorinaldi — CHANGES_REQUESTED** [review]
Hey @Jin1L mostly all looks good. I noted some small changes. Also could you add either a unit test or a screen shot to the PR to verify that the node is acting properly?

**Comment by Jin1L** [comment]
These are the screenshots to the PR.
Screenshot 2023-06-21 at 11 55 01 AM
Screenshot 2023-06-21 at 11 56 35 AM
Screenshot 2023-06-21 at 11 56 45 AM

**Review by scorinaldi — APPROVED** [review]
LGTM! thanks!

**Pull request merged**